Ping/function.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Why ping? isn't this the same as `info`?

Reply:
Ping will be only a ping API, Info instead will perform a healthcheck. For now Info doesn't check the redis connection

Reply:
Ok I see the point.

Then we could have both: 
* `GET /info`, which performs healthcheck and returns 200 with a small payload (service name and version)
* `HEAD /ping` which return 204 and no payload, to minimise the used bandwidth 

Thoughts? cc @gunzip

Reply:
yup